src/views/parents/l10n.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parents.introOne": "Scratch is a programming language and an online community where children can program and share interactive media such as stories, games, and animation with people from all over the world. As children create with Scratch, they learn to think creatively, work collaboratively, and reason systematically.",
+    "parents.introTwo": "Scratch is designed and maintained by the Lifelong Kindergarten group at the MIT Media Lab.",
+    "parents.introAbout": "About Scratch",
+    "parents.introEducators": "Info for educators",
+    "parents.learning": "Learning",
+    "parents.learningDescription": "For a one-page overview of what young people learn with Scratch, see <a href=\"https://llk.media.mit.edu/scratch/Learning-with-Scratch.pdf\">Learning with Scratch</a>. Read an article on the <a href=\"http://www.edutopia.org/kindergarten-creativity-collaboration-lifelong-learning\">creative learning approach</a>.",
+    "parents.community": "Community",
+    "parents.communityDescription": "We ask all participants on the site to follow the <a href=\"/guidelines\">Community Guidelines</a>. We do not make private account information available to anyone. For more information, please see the <a href=\"/privacypolicy\">Privacy Policy</a>.",
+    "parents.ageRange": "What is the age range for Scratch?",
+    "parents.ageRangeDescription": "While Scratch is primarily designed for 8 to 16 year olds, it is also used by people of all ages, including younger children with their parents.",
+    "parents.onlineCommunity": "What is the Scratch online community?",
+    "parents.onlineCommunityDescription": "When participating in the Scratch online community, members can explore and experiment in an open learning community with other Scratch members from all backgrounds, ages, and interests. Members can share their work, get feedback, and learn from each other.",
+    "parents.privacyPolicy": "What is your privacy policy?",
+    "parents.privacyPolicyDescription": "To protect children's online privacy, we limit what we collect during the signup process, and what we make public on the website. We don't sell or rent account information to anyone. You can find out more about our privacy policy on our <a href=\"/privacypolicy\">Privacy Policy</a> page.",
+    "parents.parentsSaying": "Learn More About Scratch",
+    "parents.parentsSayingDescription": "We often receive emails from parents thanking us for Scratch. Here are some examples:",
+    "parents.resources": "What resources are available for learning Scratch?",
```

Comment:
I think it would make more sense if the strings were stored in the order they appear on the page.